### ISSUE TITLE
Abnormal eye contact

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#8024]
**Preferred term label:**
Abnormal eye contact

**Synonyms**

**Definition (free text, please give PubMed ID)**
_will discuss at next workshop_

**Parent term (use hpo.jax.org/app)**
HP:0000758
Abnormal nonverbal communicative behavior

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
@bcoleman96 
added with label Abnormal eye contact behavior
and with preliminary definition
Any abnormality of eye contact behavior. We define eye contact as a form of nonverbal communication between two individuals who are facing each other in which an individual directs the eyes towards the eyes or face of the other individual. Eye contact occurs frequently and voluntarily during face-to-face verbal communication. The duration and frequency of eye contact that are interpreted to be appropriate may follow social and situational norms.
